calico/operations/upgrading/kubernetes-upgrade.mdx

```diff
@@ -53,6 +53,12 @@ ownership of the helm resources to the new chart location.
    kubectl patch clusterrolebinding tigera-operator tigera-operator --type=merge -p '{"metadata": {"annotations": {"meta.helm.sh/release-namespace": "tigera-operator"}}}'
    ```
 
+1. Apply the {{version}} CRDs:
+
+   ```bash
+   kubectl replace -f {{manifestsUrl}}/manifests/operator-crds.yaml
```

Comment:
`kubectl replace` will fail to create new resources. If any new CRDs exist in the YAML, then we will see an error similar to:

`Error from server (NotFound): error when replacing "{{manifestsUrl}}/manifests/operator-crds.yaml": customresourcedefinitions.apiextensions.k8s.io "*.crd.projectcalico.org" not found`

One way to ensure that existing CRDs are updated and new ones are created, would be to use:
`kubectl apply --server-side --force-conflicts -f {{manifestsUrl}}/manifests/operator-crds.yaml`

Reply:
Thank you @dimitri-nicolo, I will edit and push those changes.  Would that same change be appropriate for the CRD update step in the "All other Upgrades" section?  Happy to change that as well.

Reply:
@misterjacko yes, I believe so. If you could also add those changes that would be great. Thanks!